py/README.md

```diff
@@ -0,0 +1,397 @@
+
+Normally you should be able to call [`materialize_cohort`](#materialize_cohort) instead
+of this function, but you may use this function to get better control over
+when requests to retrieve results from the server are executed.
+
+#### `materialize_cohort_page` parameters
+Name | Required? | Description
+---- | --------- | -----------  
+`request` | Yes | A [MaterializeCohortRequest](#materializecohortrequest) indicating what cohort to materialize, what filtering and ordering criteria to apply, what fields to retrieve, and what pagination token to use (for subsequent requests.)
+ 
+
+### MaterializeCohortRequest
+
+When you want to materialize data about a cohort you've defined in workbench, you construct
+a [MaterializeCohortRequest object](swagger_docs/MaterializeCohortRequest.md). 
+
+You will later pass this request to [materialize_cohort](#materialize_cohort) if you wish
+to retrieve all the results as a generator of dictionaries, or you can pass it to 
+[`materialize_cohort_page`](#materialize_cohort_page) if you wish to make server calls
+to retrieve paginated responses containing results, one page at a time.
+
+#### `MaterializeCohortRequest` fields
+
+Name | Required? | Description
+---- | --------- | -----------
+`cohort_name`|Yes|The name of a cohort you defined in workbench. (Make sure to update any references in notebooks if you change the cohort's name.)
+`field_set`|Yes|A [FieldSet](#field-sets) indicating what data you want to retrieve about the cohort.
+`status_filter`|No|A list of [CohortStatus](swagger_docs/CohortStatus.md) values indicating cohort review statuses to filter the participants whose data is returned. By default, `[INCLUDED, NEEDS_FURTHER_REVIEW, NOT_REVIEWED]` will be used -- everything except `EXCLUDED` (participants that have been explicitly excluded.) There is no need to set this for cohorts which have not been reviewed. 
+`page_size`|No|The number of results to return in a single request to the server. Defaults to 1000. Depending on the size of data returned, you may try increasing or decreasing this to improve performance, but you generally should not have to set this.
+`page_token`|No|A pagination token used to retrieve additional results from the server after the first request. You will only need to set this if you use the [materialize_cohort_page][#materialize_cohort_page] function repeatedly to retrieve multiple pages of data explicitly.
+
+#### Field sets
+
+Field sets represent what data you want to retrieve about a cohort. 
+
+A [FieldSet object](swagger_docs/FieldSet.md) must have either its 
+[`table_query`](#table-queries) or [`annotation_query`](#annotation-queries) 
+field populated. Details for table queries and annotation queries follow.
+
+#### Table queries
+
+A [TableQuery object](swagger_docs/TableQuery.md) is used to 
+specify how to retrieve data about the cohort from a table with 
+a person_id column in the CDR version associated with the workspace housing 
```

Comment:
CDR is not defined here - also I'm pretty sure it's a term we invented. We should figure out the right user-facing language on this and make sure this gets updated everywhere (but for now, probably just define it).

Reply:
Done.